docs/technical_reference/unit_models/electrolyzer.rst

```diff
@@ -0,0 +1,245 @@
+   * does not determine equilibrium of electrolysis products in solution
+   * does not consider undesired reactions
+   * conversion of reactants is predicted by a yield model, therefore rate laws and limitations of reaction and mass transfer are not evaluated
+
+.. index::
+   pair: watertap.unit_models.electrolyzer;electrolyzer
+
+.. currentmodule:: watertap.unit_models.electrolyzer
+
+Introduction
+------------
+This model was primarily motivated to simulate the chlor-alkali membrane electrolysis process, a conventional electrolyzer for the production of chlorine gas and hydroxide (Bommaraju, 2015) (Kent, 2007). The model has been demonstrated for the chlor-alkali configuration in the electrolyzer testing file and may be generalizable to other electrolysis mechanisms, but has not been validated for generalized processes.
+
+Given a membrane electrolyzer, the catholyte and anolyte are separated by an ion exchange membrane. This provides two distinct control volumes to perform model calculations. The basis for determining electrolyzer performance is accounting for the Faradaic conversion of species with respect to the electrolysis reactions at the cathode and anode. Faradaic conversion considers equating the supplied electrical current to the amount of electrons available for electrochemical reaction by Faraday's law. The calculation of Faradaic conversion is described in "electrons passed between anode and cathode contributing to reactions" found in the :ref:`Equations <electrolyzer_equations>`.
+
+Given that Faraday's law with an applied current efficiency governs electrochemical conversion, inherently no reaction and mass transfer rate laws are considered. In this yield model, the surface area for electrochemical reaction on each electrode is the only relevant sizing variable required. This is used to validate that the current densities of each are tolerable by the material, and it is typically used as the fixed variable where the corresponding area is calculated as a degree of freedom. With no other sizing variables, effects in increased resistance through the electrolytes with further spaced electrodes and similar size dependent phenomena are not considered in the model.
+
+Degrees of Freedom
+------------------
+For a given electrolyzer design, there are 8 degrees of freedom in addition to the inlet state variables (i.e., temperature, pressure, component flowrates) that should be fixed for the model to be fully specified. In typical design cases the following 8 variables are fixed:
+
+   * membrane current density, :math:`i_{mem}`
+   * anode current density, :math:`i_{ano}`
+   * anode overpotential, :math:`\eta_{ano}`
+   * cathode current density, :math:`i_{cat}`
+   * cathode overpotential, :math:`\eta_{cat}`
+   * current, :math:`I`
+   * current efficiency, :math:`\theta_{current}`
+   * voltage efficiency, :math:`\theta_{voltage}` *or* resistance, :math:`R`
+
```

Comment:
I think there are redundant "efficiency" parameters here (considering they are all fixed so the redundancy is likely neglected).  Try clarify what is each efficiency accounting for?  For example, the current efficiency may account for all "side" electron consumption except desired electrolysis; this makes sense to parameterize since the model is not writing those side effects out.  However, for the voltage, you have already written out over potentials, ohmic potential drops, and thermo potential (you'll probably need a membrane potential"; with these you've already parameterized all "side" potential drops and what is the efficiency for?

Reply:
@lbibl This is a good catch. It should be unnecessary given the new addition of `eq_voltage_cell`. The only value to this is if a user doesn't know the anode and cathode overpotentials and/or the resistance. Then a 'redundant' equation can replace

```
b.voltage_cell == (
                b.voltage_reversible
                + b.anode_overpotential
                + b.cathode_overpotential
                + b.current * b.resistance
            )
```
with
```
b.voltage_reversible == b.voltage_cell * b.efficiency_voltage
```
where 
```
b.efficiency_voltage = (
                + b.anode_overpotential
                + b.cathode_overpotential
                + b.current * b.resistance
)
```
because the individual values are unknown.

Do you think this should just be removed or added as a config option?

Reply:
@avdudchenko also could use some input based on the intended analysis around the model

Reply:
I recommend removing the voltage efficiency.